demos/simple-storage/repro-build-protocol/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@web3api/examples-simple-storage-repro",
+  "version": "0.0.1-prealpha.1",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "build": "w3 build",
+    "test:env:up": "w3 test-env up && yarn contracts:deploy",
+    "test:env:down": "w3 test-env down",
+    "reproducible-build": "w3 reproducible-build build --quiet=false"
```

Comment:
I feel like the CLI command for this should be reworked to something more intuitive. Maybe `w3 build --reproducible`? Or maybe it should even be the default `w3 build` and a local build should be `w3 build --local`. But probably something should change here.